examples/meta/src/neural_nets/feedforward_net_classification.sg

```diff
@@ -0,0 +1,45 @@
+CSVFile f_feats_train("../../data/fm_train_real.dat")
+CSVFile f_feats_test("../../data/fm_test_real.dat")
+CSVFile f_labels_train("../../data/label_train_twoclass.dat")
+
+#![create_features]
+RealFeatures features_train(f_feats_train)
+RealFeatures features_test(f_feats_test)
+BinaryLabels labels_train(f_labels_train)
+#![create_features]
+
+#![add_layers]
```

Comment:
@karlnapf Even after the changes by @lisitsyn, it gives segfault.
```
auto num_feats = features_train->get_num_features();
auto layers = some<CNeuralLayers>();
layers = layers->input(num_feats);
layers = layers->linear(50);
layers = layers->softmax(2);
auto all_layers = layers->done();
```
Above is the `cpp` translation. Segmentation fault occurs in the 4th line.